apps/web/src/hooks/useNovuAPI.ts

```diff
@@ -42,24 +43,27 @@ export const useTelemetry = () => {
     api.postTelemetry(event, data)
   );
 
-  return (event: string, data?: Record<string, unknown>) => {
-    const mixpanelEnabled = !!process.env.REACT_APP_MIXPANEL_KEY;
+  return useCallback(
+    (event: string, data?: Record<string, unknown>) => {
+      const mixpanelEnabled = !!process.env.REACT_APP_MIXPANEL_KEY;
 
-    if (mixpanelEnabled) {
-      const segmentDeviceId = localStorage.getItem('ajs_anonymous_id');
-      const userId = localStorage.getItem('ajs_user_id');
-      if (userId) {
-        mixpanel.identify(userId);
-      }
-      mixpanel.register({ $device_id: segmentDeviceId });
-      const sessionReplayProperties = mixpanel.get_session_recording_properties();
+      if (mixpanelEnabled) {
```

Comment:
Prettier issues?

Reply:
i think its because of the callback wrapping it, so extra space